dashboard/app/controllers/admin_search_controller.rb

```diff
@@ -52,8 +52,10 @@ def search_for_teachers
 
       # Remove newlines from the full_address field, replacing them with spaces.
       @teachers.each do |teacher|
-        teacher[3].gsub!("\r", ' ')
-        teacher[3].gsub!("\n", ' ')
+        if teacher[3].present?
```

Comment:
As written it cannot be, since the columns are being plucked rather than selected, giving an array rather an ActiveRecord bobject.